### ISSUE TITLE
Set up Saucelabs for testing

### ISSUE THREAD

**Issue by marriej** [department-of-veterans-affairs/va.gov-team#7254]
Using these instructions, to set up SauceLabs.

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/quality-assurance/cross-platform-testing-plan.md#browsers-supported

Please add comment with whether you were successful or not and re-assign in the following order:

- [ ] Bridget

- [ ] Eric

- [ ] Alexis

- [ ] Ambika

- [ ] Dan

- [ ] Patrick

**Comment by bridgethapner** [comment]
I logged in and st up the proxy. I'm stuck on [this step](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/platform/quality-assurance/cross-platform-testing-plan.md#launching-the-proxy).

**Comment by ericbuckley** [comment]
@marriej is there any more information from the VSP team on what we should be using Sauce Labs for?  From what I can tell there are 2 possible use cases.

1) we can use it to manually run tests in browsers, say for testrails acceptance, that aren't available on our laptops (like mobile phones, or IE)

2) we can use it to run the vets-website end-to-end integration tests

If the latter it seems like the bigger ask from VSP would be to create end-to-end tests for the features we are planning on releasing.

**Comment by ericbuckley** [comment]
@marriej based on Peter's feedback in DSVA, this isn't a separate ticket from #6835.  When we run through all of our test cases in different browsers, we *can* use Saucelabs to verify that it will run correctly in our desired browsers if for say our machines don't have those browsers installed.

So for example, if you have a macbook and are need to run test case #1 in IE11, you can sign into Saucelabs, select IE11, and manually run the test there to make sure it works as expected.

tldr; Saucelabs is a tool for testing, not a requirement
